### PR TITLE
Amp stories best practices update 

### PR DESCRIPTION
**Pull request by CrystalOnScript**
This PR is waiting on a formatting fix on asset behavior in tables.

### CODE REVIEW

**Comment by keevkeev** [comment]
@CrystalOnScript sorry about the delay on the fix, I haven't forgotten about it. I just need to fix a few other bugs on the branch where this is located. I'll let you know when it's merged.

**Comment by CrystalOnScript** [comment]
Awesome, thank you! The AMP Stories team would like this live before April 5th - does that seem possible?

**Comment by keevkeev** [comment]
@CrystalOnScript as long as there are no other bugs you've found, the image fixed is merged into future and this should be GTG. Let me know if you find anything else

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_story_best_practices.md [hunk 1]
```diff
@@ -7,38 +7,384 @@ formats:
...
+    <td>{{ image('/static/img/docs/guides/storiesbp/embed-do.jpg', 250, 500, layout='intrinsic', alt='Example of well used embed on AMP story page', align='center' ) }}</td>
+    <td>{{ image('/static/img/docs/guides/storiesbp/embed-dont.jpg', 250, 500, layout='intrinsic', alt='AMP story page with no background or header, but a single embed looks unfinished', align='center' ) }}</td>
+  </tr>
+  <tr>
+    <td>The embed on this page integrates well with the rest of the layout. The headline, date, and background graphics help enhance the visual.</td>
+    <td>Putting the embed alone on the page looks unfinished and does not integrate with the full story well.</td>
+  </tr>
+</table>
+
+### Add Additional Content with Attachments
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_story_best_practices.md]: Inconsistent title casing.

**Comment by sebastianbenz** [comment]
(I assuming that the remaining GIFs will be switched to mp4s in a different PR)

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_story_best_practices.md (hunk 1 above)
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/amp_story_best_practices.md]: good catch, thanks!

**Comment by CrystalOnScript** [comment]
Fixed gif! Merging

**Pull request merged**